client/web/src/lib/ColorPicker.ts

```diff
@@ -0,0 +1,46 @@
+interface ColorPickerOptions {
+  el: Element;
+};
+
+export interface RGBAColor {
+  r: number;
+  g: number;
+  b: number;
+  a?: number;
```

Comment:
`a` will always be included in every circumstance.

Reply:
Updated